### PR TITLE
Add Dragino CPL03-LB connector with logo and payload configuration

### PR DESCRIPTION
**Pull request by Freddyminu**
## Decoder Description

Added decoder code for CPL03-LB.
https://www.dragino.com/products/lorawan-nb-iot-door-sensor-water-leak/item/278-cpl03-lb-ls.html

## Type of change

- [X] Adding a new Decoder of Connector
- [ ] Adding a new Decoder of Network
- [ ] Update or fixing an issue in existing Decoder

## Decoder Information and Payload to test and review

- [ ] Documentation of the hardware or protocol:
- [X] Payload of example the test the decoder:

## Checklist for Adding a New Decoder

- [X] Created a new folder under `./decoders/network/` or `./decoders/connector/` with the name of your decoder.
- [X] Added a `network.jsonc` or `connector.jsonc` file that follows the structure defined in `./schema/`.
- [X] Created version folders and added `manifest.jsonc` files for each version.
- [X] Followed the folder structure guidelines for manufacturer and sensor/device model.
- [X] The code has unit test and it's in TypeScript.

## Additional Notes

Please add any other information that you think is important.

### CODE REVIEW

**Review by FabianoEger — APPROVED** [review]

File: decoders/connector/dragino/cpl03lb/v1.0.0/payload.test.ts [hunk 1]
```diff
@@ -0,0 +1,186 @@
+/* eslint-disable unicorn/numeric-separators-style */
+import { describe, expect, test } from "vitest";
+
+import { decoderRun } from "../../../../../src/functions/decoder-run";
+
+const file_path =
+  "decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts" as const;
+
+function preparePayload(payloadHex, port) {
```
Comment (mateuscardosodeveloper) [inline decoders/connector/dragino/cpl03lb/v1.0.0/payload.test.ts:9]: Add type annotation to the function.

File: decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts [hunk 2]
```diff
@@ -0,0 +1,235 @@
...
+/* eslint-disable prefer-destructuring */
+/* eslint-disable prettier/prettier */
+/* eslint-disable no-bitwise */
+/*
+ * CPL01
+ */
+// Add ignorable variables in this array.
+const ignore_vars = [] as any;
+
+function toTagoFormat(object_item, group, prefix = "") {
```
Comment (mateuscardosodeveloper) [inline decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts:13]: Add type annotation to the function.

File: decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+        value: object_item[key],
+        group,
+      });
+    }
+  }
+
+  return result;
+}
+
+function getzf(c_num) {
```
Comment (mateuscardosodeveloper) [inline decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts:39]: Add type annotation to the function.

File: decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts [hunk 4]
```diff
@@ -0,0 +1,235 @@
...
+  if (parseInt(c_num) < 10) c_num = `0${c_num}`;
+
+  return c_num;
+}
+
+function getMyDate(str) {
```
Comment (mateuscardosodeveloper) [inline decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts:45]: Add type annotation to the function.

File: decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts [hunk 5]
```diff
@@ -0,0 +1,235 @@
...
+  const c_Min = c_Date.getMinutes();
+  const c_Sen = c_Date.getSeconds();
+  const c_Time = `${c_Year}-${getzf(c_Month)}-${getzf(c_Day)} ${getzf(
+    c_Hour
+  )}:${getzf(c_Min)}:${getzf(c_Sen)}`;
+
+  return c_Time;
+}
+
+function datalog(i, bytes) {
```
Comment (mateuscardosodeveloper) [inline decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts:63]: Add type annotation to the function.

File: decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts [hunk 6]
```diff
@@ -0,0 +1,235 @@
...
+  );
+  var string;
+  if (aa == "CPL01")
+    string = `[${aa},${bb},${cc},${dd},${ee},${ff},${gg},${hh},${ii},${jj}]`;
+  else string = `[${aa},${bb},${cc},${gg},${hh},${ii},${jj}]`;
+  string = string.concat(",");
+  return string;
+}
+
+function Decoder(bytes, port) {
```
Comment (mateuscardosodeveloper) [inline decoders/connector/dragino/cpl03lb/v1.0.0/payload.ts:97]: Add type annotation to the function.

**Review by mateuscardosodeveloper — CHANGES_REQUESTED** [review]

**Review by mateuscardosodeveloper — APPROVED** [review]

**Pull request merged**